backend/server/public/src/actions.js

```diff
@@ -106,6 +117,25 @@ export const getOneAd = (ad_id, url = "/facebook-ads/ads") => {
   };
 };
 
+export const getGroupedAttrs = (
+  groupingKind = "advertisers",
+  root_url = "/facebook-ads/ads"
+) => {
+  let path = `${root_url}/${groupingKind}`;
+  return (dispatch, getState) => {
+    let state = getState();
+    dispatch(requestingRecentGroupedAttr());
```

Comment:
Do we need this if it doesn't actually change any state? See the note in reducers below.